Resources/Prototypes/Entities/Objects/Specific/Security/target.yml

```diff
@@ -0,0 +1,149 @@
+- type: entity
+  id: BaseTarget
+  parent: BaseStructureDynamic
+  abstract: true
+  components:
+  - type: Sprite
+    sprite: Objects/Specific/Security/target.rsi
+    state: target_stake
+    noRot: true
+    netsync: false
+  - type: Repairable
+  - type: DamagePopup
+    damagePopupType: Combined
+  - type: Fixtures
+    fixtures:
+    - shape:
+        !type:PhysShapeCircle
+        radius: 0.35
+      density: 200
+      mask:
+      - FullTileMask
+      layer:
+      - WallLayer
+  - type: InteractionOutline
+  - type: Physics
+  - type: Damageable
+    damageContainer: Inorganic
+    damageModifierSet: Metallic
```

Comment:
While it might make sense for it to be metallic, this will skew all the damage stats people would infer from testing using the dummy. So it should probably just have no modifier.